src/i18n/workspaces.cs.json

```diff
@@ -36,6 +36,7 @@
   "publishWorkspaceSuccess": "Pracovní prostor byl publikován",
   "publishWorkspaceError": "Nepodařilo se publikovat pracovní prostor",
   "workspacePRCreated": "Pracovní prostor byl publikován, pull request vytvořen:",
+  "workspaceMayBeDeleted": "Nyní může být pracovní prostor smazán bez ztráty dat. Diagramy a další přílohy budou uloženy.",
```

Comment:
Mohli bychom vypíchnout, že smazáním umožní uživatel ostatním uživatelům přispívat k daným slovníkům. Ale zas aby je to pak naopak neodradilo. No nevím.

Reply:
Nebo přesněji: 

Po schválení pull requestu a jeho zapracování do hlavní větve projektu může být pracovní prostor smazán bez ztráty dat. Diagramy a další přílohy budou uloženy.

Reply:
Fixed